### PR TITLE
Update macOS installation with MacPorts.

### PR DESCRIPTION
**Pull request by conradwt**
This PR adds the following:

- renames OS X to macOS

- adds MacPorts installation information (https://github.com/macports/macports-ports/pull/2163)

### CODE REVIEW

File: getting-started/quick-start.md [hunk 1]
```diff
@@ -20,15 +20,25 @@ If you already have `crystal`, `amber`, and a database installed, you can [skip
...
+- Homebrew
 
-```text
-brew install amber
-```
+  ```text
+  brew install amber
+  ```
+
+  Note: This also installs the Crystal Programming Language.
```
Comment (robacarp) [inline getting-started/quick-start.md]: this note doesn't need to be repeated, but could be provided above, near the title of the section.

File: getting-started/quick-start.md [hunk 2]
```diff
@@ -65,9 +75,9 @@ Amber works with `postgresql` \(default\), `mysql`, or `sqlite`.
 
 If you don’t already have one of these installed, please follow the guides provided by each database maintainer:
 
-* [Postgresql Installation Guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides)  
-* [MySQL Installation Guides](https://dev.mysql.com/doc/refman/8.0/en/installing.html)  
-* [Tutorial on installing SQLite](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm)  
+- [Postgresql Installation Guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
+- [MySQL Installation Guides](https://dev.mysql.com/doc/refman/8.0/en/installing.html)
+- [Tutorial on installing SQLite](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm)
```
Comment (robacarp) [inline getting-started/quick-start.md]: Why change all of these * to -? That's adding a lot of lines to your pull that are unrelated to the purpose of it.

File: getting-started/quick-start.md (hunk 1 above)
Comment (conradwt) [reply getting-started/quick-start.md]: Will update.

File: getting-started/quick-start.md (hunk 2 above)
Comment (conradwt) [reply getting-started/quick-start.md]: VS Code and Github defaults to CommonMark standard for Markdown syntax.  Is there a particular reason that you're using `*` for unordered lists instead `-`?
